### PR TITLE
Updated node version to 18

### PR DESCRIPTION
**Pull request by NehaNaithani**
[sc-16022]

### CODE REVIEW

File: Dockerfile.dev [hunk 1]
```diff
@@ -1,10 +1,10 @@
-FROM node:gallium-buster AS builder
+FROM node:hydrogen-buster AS builder
 
 ARG PKG_VERSION
 
 WORKDIR /tmp
 
-RUN npm config set unsafe-perm true
+# RUN npm config set unsafe-perm true
```
Comment (baghbidi) [inline Dockerfile.dev]: Please delete this line is the `npm` command is not needed

File: Dockerfile.dev [hunk 2]
```diff
@@ -34,11 +34,11 @@ RUN pip3 install --no-cache --upgrade pip setuptools
 
 # https://stackoverflow.com/questions/52196518/could-not-get-uid-gid-when-building-node-docker
 # Workaround until we fix our node alpine image
-RUN npm config set unsafe-perm true
+# RUN npm config set unsafe-perm true
```
Comment (baghbidi) [inline Dockerfile.dev]: ditto

File: Dockerfile.rel [hunk 3]
```diff
@@ -7,7 +7,7 @@ WORKDIR /tmp
 # Required by node-gyp
 RUN apt install python3
 
-RUN npm config set unsafe-perm true
+# RUN npm config set unsafe-perm true
```
Comment (baghbidi) [inline Dockerfile.rel]: Same here

File: Dockerfile.rel [hunk 4]
```diff
@@ -21,11 +21,11 @@ RUN npm version $PKG_VERSION --allow-same-version
 
 RUN npm pack
 
-FROM node:gallium-alpine3.15
+FROM node:hydrogen-alpine3.17
 
 # https://stackoverflow.com/questions/52196518/could-not-get-uid-gid-when-building-node-docker
 # Workaround until we fix our node alpine image
-RUN npm config set unsafe-perm true
+# RUN npm config set unsafe-perm true
```
Comment (baghbidi) [inline Dockerfile.rel]: And here

File: src/data/migrations/20181113094807-add-missing-constraints.js [hunk 5]
```diff
@@ -126,21 +134,21 @@ module.exports = {
 
       // return queryInterface.removeConstraint('Flows', 'userId')
```
Comment (baghbidi) [inline src/data/migrations/20181113094807-add-missing-constraints.js]: Looks like this whole block is not needed, right? If so, please delete.

**Review by baghbidi — CHANGES_REQUESTED** [review]

**Review by baghbidi — COMMENTED** [review]
LGTM

File: Dockerfile.dev (hunk 1 above)
Comment (NehaNaithani) [reply Dockerfile.dev]: Yes this command is not needed now

File: src/data/migrations/20181113094807-add-missing-constraints.js (hunk 5 above)
Comment (NehaNaithani) [reply src/data/migrations/20181113094807-add-missing-constraints.js]: Removed

File: Dockerfile.rel (hunk 4 above)
Comment (NehaNaithani) [reply Dockerfile.rel]: Removed

File: Dockerfile.rel (hunk 3 above)
Comment (NehaNaithani) [reply Dockerfile.rel]: Removed

File: Dockerfile.dev (hunk 2 above)
Comment (NehaNaithani) [reply Dockerfile.dev]: Removed

**Review by baghbidi — APPROVED** [review]

**Pull request merged**